### PR TITLE
Add user management to manager node.

### LINKED ISSUE #236: [mainnet-manager] Add user management to manager node.
> Related to: #227

### Requirements
- Login (password protection to serve the manager on the web) + logout endpoint.
- Change password endpoint (dont let the user use other endpoints if he hasnt changed the default pw first).

### PR DESCRIPTION
**Pull request by evanlinjin**
closes #236

### CODE REVIEW

**Comment by evanlinjin** [comment]
**To Do:**
- [x] Create endpoints: change-password, init-admin.
- [x] Default values.
- [x] Tests.

File: cmd/manager-node/commands/root.go [hunk 1]
```diff
@@ -1,63 +1,86 @@
...
+	Use:   "manager-node [config-path]",
 	Short: "Manages Skywire App Nodes",
-	PreRun: func(_ *cobra.Command, _ []string) {
-		if pk.Null() && sk.Null() {
-			pk, sk = cipher.GenerateKeyPair()
-			log.Println("No keys are set. Randomly generating...")
+	Run: func(_ *cobra.Command, args []string) {
+
+		var configPath string
+		if len(args) == 0 {
```
Comment (ivcosla) [inline cmd/manager-node/commands/root.go:58]: You can specify instead MinArgs in cobra.Command

File: cmd/manager-node/commands/root.go [hunk 2]
```diff
@@ -1,63 +1,86 @@
...
+		} else {
+			configPath = args[0]
 		}
-		cPK, err := sk.PubKey()
-		if err != nil {
-			log.Fatalln("Key pair check failed:", err)
+		log.Infof("config path: '%s'", configPath)
+
+		var config manager.Config
+		config.FillDefaults()
```
Comment (ivcosla) [inline cmd/manager-node/commands/root.go:69]: nice, I think we have used `Ensure` before for that behaviour

File: cmd/manager-node/commands/root.go (hunk 1 above)
Comment (evanlinjin) [reply cmd/manager-node/commands/root.go:58]: That won't work. 0 args is valid if `findConfigPath()` function succeeds.

File: cmd/manager-node/commands/root.go (hunk 2 above)
Comment (evanlinjin) [reply cmd/manager-node/commands/root.go:69]: `FillDefaults` to `Ensure`. But isn't `FillDefaults` more descriptive?

Comment (ivcosla) [reply cmd/manager-node/commands/root.go:69]: I really find it more descriptive yes, just a matter of being consistent with our naming. Can rename the other config method instead.

File: cmd/manager-node/commands/root.go (hunk 1 above)
Comment (ivcosla) [reply cmd/manager-node/commands/root.go:58]: then all good

**Review by ivcosla — APPROVED** [review]
Great job, and very clean code!.

**Pull request merged**